apps/frontend/src/renderer/components/github-prs/components/PRFilesViewer.tsx

```
@@ -0,0 +1,317 @@
+                key={fileKey}
+                className="rounded-md border border-border/50 overflow-hidden"
+              >
+                {/* File header */}
+                <button
+                  onClick={() => hasDiff && toggleFile(file.path)}
+                  className={cn(
+                    'w-full flex items-center gap-2 px-3 py-2 bg-muted/30 hover:bg-muted/50 transition-colors',
+                    hasDiff && 'cursor-pointer',
+                    !hasDiff && 'cursor-default'
+                  )}
+                >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)